### PR TITLE
Husky configuration update for husky 5

### PR DESCRIPTION
**Pull request by svenjens**
Description 
This pr changes the husky configuration so its working with husky 5 precommit. 

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1 @@
+npx --no-install lerna run --concurrency 1 --stream precommit
```
Comment (afjlambert) [inline .husky/pre-commit:1]: Just curious; why did you add `npx --no-install ...`?

**Review by afjlambert — CHANGES_REQUESTED** [review]
For me the hooks did not work after a `npm run cleaninstall`. I had to `npx husky install` first. We could add that to the docs, but maybe better is following the docs. It states we can add this to our `package.json`:
```
"scripts": {
    "postinstall": "husky install"
  }
```
See https://typicode.github.io/husky/#/

File: .husky/_/husky.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (afjlambert) [inline .husky/_/husky.sh]: I think you can remove this husky.sh file. Why did you add this?
Reading the docs, I think it should be installed automatically in `.git/hooks`-folder after you `npx husky install`.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: .husky/_/husky.sh (hunk 2 above)
Comment (svenjens) [reply .husky/_/husky.sh]: This is the file that is added by installing husky. 

Husky created it. :-)

File: .husky/pre-commit (hunk 1 above)
Comment (svenjens) [reply .husky/pre-commit:1]: Its running Lerna without installing in.
If we don't add npx --no-install it can't find lerna.

File: .husky/_/husky.sh (hunk 2 above)
Comment (afjlambert) [reply .husky/_/husky.sh]: Is this for Windows users maybe? Seems to run fine without.
I think I installed husky in a clean project and it did not add this file 🤔

**Comment by svenjens** [comment]
Let me check if I delete this file it still works

Op ma 1 mrt. 2021 om 12:34 schreef André Lambert <notifications@github.com>

**Comment by robinpiets** [comment]
I'm getting massive change requests after running `npm i` in the root. 

image

**Review by robinpiets — APPROVED** [review]
👍🏽

**Comment by svenjens** [comment]
> 👍🏽

@robinpiets  I see you added the husky .gitignore. I didn't get that file on install..
